orm_lib/inc/drogon/orm/Row.h

```diff
@@ -77,11 +77,13 @@ class DROGON_EXPORT Row
     ConstReverseIterator rend() const;
     ConstReverseIterator crend() const;
 
+    Row() noexcept = default;
     Row(const Row &r) noexcept = default;
     Row(Row &&) noexcept = default;
+    Row &operator=(const Row &) = default;
 
   private:
-    const Result result_;
+    Result result_;
```

Comment:
Why do we need to remove the const keyword?

Reply:
@an-tao The copy assignment operator is **_implicitly_** deleted by const members, but it is required by `std::future<Row>` along with the default constructor. If we **_explicitly_** define some assignment operators or constructors (i.e., `Row(const Row &r) noexcept = default;` and `Row(Row &&) noexcept = default;`), the other undefined assignment operators and constructors will be **_implicitly_** deleted. Since `std::future<Row>` also needs the default constructor, we must define them **_explicitly_** as well.

Reply:
OK